README.md

```diff
@@ -2,23 +2,53 @@
 
 [View it live ❯](https://developers.login.gov/)
 
-Run the server locally:
+## Development
 
+### Docker
+
+Build and run the site
+
+```sh
+docker-compose build
+docker-compose up
+open http://localhost:4000
 ```
-make run
+
+Run tests
+
+```sh
+docker-compose run web bundle exec rspec spec
 ```
 
-Run the tests
+Some changes, like `_config.yml` don't get picked up automatically in Docker. You can force a re-build of the site without re-building the whole container:
 
+```sh
+docker-compose run web bundle exec jekyll build
 ```
-make test
+
+You can drop into a bash shell in the container with 
+
+```sh
+docker-compose run web bash
 ```
 
-## Updating the template
+### Locally
 
-This site uses the [U.S. Web Design Standards](https://standards.usa.gov). To update them:
+Run the server locally: 
 
-1. Download the [latest release](https://standards.usa.gov/download)
```

Comment:
I didn't have any styles in the local site until I grabbed uswds via npm and copied stuff over per these instructions.

Reply:
🤷‍♂ I took out the `RUN npm run build-uswds` and have the styles somehow, but I may have a dirty environment....going to do a git clean and try again

Reply:
Ok! So with a clean environment, the styles didn't load automatically.

But then I did this:
```
docker-compose exec web bash
```
and ran `make build` inside the container, and things worked. So maybe the default `up` is missing something that `make build` or `make run` do?

Reply:
OK, that makes sense. Checking out that other PR and will run it all in a fresh env